hw3/03-react/game-of-thrones/src/views/Home.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+class Home extends Component {
+  render() {
+    return (
+      <h1>
+        <center>Welcome to our Application!</center>
```

Comment:
[The center tag is deprecated.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center) 

Also, avoid using html tags for styling, use CSS instead.